docs/src/examples/gallery.md

```diff
@@ -1,4 +1,6 @@
-# [PGFPlots manual gallery](@id manual_gallery)
+# [PGFPlots gallery](@id manual_gallery)
```

Comment:
Does this refer to pgfplots, the LaTeX package? We consistently use lower case for that in the manual.

Reply:
Now it refers to the latex package so lets make it lowercase then.

Reply:
Is this really true?

![IMG_0088](https://user-images.githubusercontent.com/1282691/71446413-557fd100-2723-11ea-8af7-3ac89fe7420b.jpeg)

Reply:
Not really. Sorry, I must have mixed up something. Good catch! Please use whatever you prefer.